tests/test_linestring.py

```diff
@@ -6,6 +6,109 @@
 from shapely.geometry import LineString, asLineString, Point, LinearRing
```

Comment:
Looks like you can remove the `lgeos` import on the line above this (github won't let me comment on that line...)

Reply:
Indeed, thanks!